### DISCUSSION TITLE
How to delete brush textures?

### DISCUSSION THREAD

**Discussion by 6xolotl** [keiwando/toonsquid#285] [Q&A] [answered]
i feel bad asking so many questions but thats kinda what a support forum is for innit

anyway, how do i delete brush textures? im trying to figure out making custom brushes, but i dont want a thousand scrapped brush tips left over after ive settled on one. how do i get rid of the ones im not gonna use?

**Comment by DinoAwakening** [comment]
You can try these options, so your brush will get back to a simple shape:

1. Shape>Set a simple shape (with no texture)
2. Grain>Set a simple square (with no texture)
3. Scatter>Set scatter to 0%

If it doesn’t work, let me now. I think I’ve mastered brushes settings enough to achieve proper results :P

**Reply by 6xolotl** [reply]
Thank you for the comment, but this wasn’t what I meant! ^^;

I was asking how to delete a brush shape from the texture library. Holding down on it didn’t work, and there’s no delete button, so I was confused—

I hope there’s a way to delete them, I don’t want a graveyard of unused brush shapes in my texture library ;w;

**Reply by DinoAwakening** [reply]
Oh, I think there’s not currently a way to do that :((

**Comment by keiwando** [comment, marked as answer]
With the recent 1.9 update you can now select and delete brush textures that you previously imported into the brush texture library.
